docs/elements/hubspot-crm/tips.md

```diff
@@ -12,7 +12,7 @@ order: 50
 
 ## Tips
 
-What should I do if I receive the following error message:
-![HubSpot Scope](img/scope.png)
+The Tips section includes more information about the element including FAQs, endpoint limitations, and general tips.
 
-Verify that the scopes selected in registered app are aligned with the permissions allowed under the selected Portal ID.
+
+{% include hubspot/tips.md %}
```

Comment:
This includes statement is failing. @KelseyStevenson you might want to double check this one. It can't find the file.